src/rebuild/scan.c

```diff
@@ -939,6 +914,43 @@ rebuild_container_scan_cb(daos_handle_t ih, vos_iter_entry_t *entry,
 	return rc;
 }
 
+bool
+is_rebuild_scanning_tgt(struct rebuild_tgt_pool_tracker *rpt)
+{
+	struct pool_target	*tgt;
+	unsigned int		idx = dss_get_module_info()->dmi_tgt_id;
+	d_rank_t		rank;
+	int			rc;
+
+	D_ASSERT(rpt != NULL);
+	D_ASSERT(rpt->rt_pool != NULL);
+	D_ASSERT(rpt->rt_pool->sp_map != NULL);
+
+	/* Let's use NULL for now, because subgroup == master group for
+	 * all of test anyway. Once we resolve the race between cart
+	 * group destroy and rebuild, it should use cart group inside
+	 * ds_pool. (DAOS-1943) FIXME
```

Comment:
is this comment still needed?
the rank used in pool map is rank in primary group, so seems no problem for the usage.

Reply:
ok, I will remove it